test_scripts/API/VehicleData/Refactor_Fuel_Information/007_VD_parameter_in_API_DB_app_version_less_than_parameter_version.lua

```diff
@@ -0,0 +1,78 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Referring to this comment https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2413#discussion_r451132043, I think this new test should be removed and the existing test `VehicleDataVersioning/003_VD_parameter_in_API_DB_app_version_less_then_parameter_version.ua` should be expanded to test for the `FuelRange` params